packages/components/psammead-story-promo-list/src/__snapshots__/index.test.jsx.snap

```diff
@@ -188,6 +188,7 @@ exports[`StoryPromo list should render correctly 1`] = `
     >
       <div
         class="c3"
+        dir="ltr"
```

Comment:
I don't think this snapshot needs to be updated in this PR.

Reply:
It's because the top level version of story-promo was previously set to an alpha version of 4 that didn't include the dir prop. Now I've updated it, this is included in the output.

Reply:
Ah, and psammead-story-promo-list includes psammead-story-promo in tests, but not as a dev dependency.

Reply:
Or do we only have one devDependencies in the psammead root package.json?

Reply:
^ We do only include devDependencies in the root